nnef/src/registry.rs

```diff
@@ -17,11 +17,35 @@ pub type Extension = Box<
         + Sync,
 >;
 
+#[derive(Clone)]
+pub struct PrimitiveDef {
```

Comment:
should if be named `PrimitiveDecl` instead of `Def` ?

Reply:
👍